lib/core/form/services/widget-visibility.service.ts

```diff
@@ -67,6 +67,9 @@ export class WidgetVisibilityService {
         const rightValue = this.getRightValue(form, visibilityObj);
         const actualResult = this.evaluateCondition(leftValue, rightValue, visibilityObj.operator);
 
+        if (!this.isFieldValueValid(form, visibilityObj)) {
+            return false;
```

Comment:
Returning a value here will block the chaining condition operator. 
I think we need to find another way to fix this problem

Reply:
The expected behaviour is that the visibility condition should fail, if any one field in the condition has an invalid value. In that case, it is unnecessary to proceed with evaluating the condition